services/avs/mgmt/2020-03-20/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name          = "azure_avs_mgmt_2020_03_20"
+version       = "0.1.0"
+edition       = "2018"
+
+[dependencies]
+serde = { version = "*", features = ["derive"] }
```

Comment:
Let's not do "*" as versions. Not only is this a bad idea, crates.io won't allow you to publish a crate with dependencies specified in this way.

Reply:
Living on the edge, @rylev. I agree. I'm want to generate these Cargo.toml files too. I've been waiting for progress from https://github.com/ctaggart/autorust/issues/35.

Reply:
I set the versions in the most recent commit.